### ISSUE TITLE
jme3-plugins loads MorphPrimitivesTest incorrectly

### ISSUE THREAD

**Issue by stephengold** [jMonkeyEngine/jmonkeyengine#2110]
I noticed a defect in GltfLoader while developing the MonkeyWrench library.

The test data is [MorphPrimitivesTest from Khronos's glTF sample models](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/MorphPrimitivesTest).

Here's how it looks in CompareLoaders when "MorphPrimitivesTest/glTF/MorphPrimitivesTest.gltf" is loaded using jme3-plugins:

![MorphPrimitivesTest](https://github.com/jMonkeyEngine/jmonkeyengine/assets/2287609/d24be9a2-9bf2-4f84-898b-99f248e61536)

I believe the best approach to solve this issue will be:
1. Add a `MorphControl` to every model with morph targets, even if there are no animation tracks.
2. Initialize the morph state of each `Geometry` by invoking `setMorphState()` with the "weights" values in the glTF.

**Comment by riccardobl** [comment]
Is this fixed by https://github.com/jMonkeyEngine/jmonkeyengine/pull/2109 ?

**Comment by codex128** [comment]
No, I've already tried MorphPrimitivesTest with #2109 and it still fails. I suspect this is because the [gltf file](https://github.com/KhronosGroup/glTF-Sample-Models/blob/master/2.0/MorphPrimitivesTest/glTF/MorphPrimitivesTest.gltf) does not explicitly define animations (unlike the MorphStressTest gltf).
